src/reporters/lang/en.js

```diff
@@ -95,6 +96,8 @@ const messages = {
   lockfileMerged: 'Merge conflict detected in yarn.lock and successfully merged.',
   lockfileConflict:
     'A merge conflict was found in yarn.lock but it could not be successfully merged, regenerating yarn.lock from scratch.',
+  lockfileModulesConflict:
+    '\n  There is already a node_modules folder and no yarn.lock.\n  This probably means you are transitioning from npm to yarn.\n  Proceeding means the node_modules folder will be overwritten by yarn.\n  Do you want to proceed? (yes/no)',
```

Comment:
We should not be adding new lines for visual aesthetics into the translatable strings. This text can also be shorter and refer people to [`yarn import`](https://yarnpkg.com/en/docs/cli/import#search). Even better would be to offer running `yarn import` instead of the current command.